decidim-core/lib/tasks/generators.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```

Comment:
Can you rename this file to be consistent with the others? It should be like `decidim-core/lib/tasks/decidim_pwa_tasks.rake`